spec/menu_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'menu'
+
+describe Menu do
+  let(:menu) { described_class.new }
+  let(:food) { 'Satay' }
+  let(:random_food) { Dish.new('Turkish Delight', 4.00) }
+
+  it 'allows customers to view the menu dishes' do
+    expect{ menu.dishes }.not_to raise_error
+  end
+
+  describe '#get_dish' do
+    it 'can look up an dish' do
+      expect(menu.get_dish(food).name).to eq food
+    end
+
+    it 'raises an error if the dish is not on the menu' do
+      expect{ menu.get_dish(random_food.name) }.to raise_error 'Sorry we don\'t have that on our menu'
```

Comment:
Metrics/LineLength: Line is too long. [102/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)